MAiD - Case Management App/force-app/main/default/objects/Form_1634__c/fields/If_Organ_Failure_other_specify__c.field-meta.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>If_Organ_Failure_other_specify__c</fullName>
+    <encryptionScheme>None</encryptionScheme>
+    <externalId>false</externalId>
+    <label>If Organ Failure other, specify</label>
+    <length>255</length>
```

Comment:
@RanadheerRG: Do we have an example of how long the failure reason could be? Having 255 characters is ok for now, but in future we could provide precise character length which the customer wants. The reason i am calling this out is because if the customer will only enter Consult Patient/Patient did not disclose, then it will not go beyond 100 characters.

Reply:
> Do we have an example of how long the failure reason could be?
No, client did not provide any example.

 

> Having 255 characters is ok for now, but in future we could provide precise character length which the customer wants. The reason i am calling this out is because if the customer will only enter Consult Patient/Patient did not disclose, then it will not go beyond 100 characters.
I understand and as per client refinement call, they are not too worried on how we test the application. Theyhad told us to enter dummy text and test it.